volume-providers/vpc/messages/messages_en.go

```diff
@@ -59,11 +66,12 @@ var messagesEn = map[string]util.Message{
 		Action:      "Volume ID is not valid. Please review provided URL for valid volume ID.",
 	},
 	"InvalidVolumeName": util.Message{
-		Code:        "InvalidVolumeName",
-		Description: "'%s' volume name is not valid. Please check https://cloud.ibm.com/docs/infrastructure/vpc?topic=vpc-rias-error-messages#validation_invalid_name",
+		Code: "InvalidVolumeName",
+		// TODO! for volume name invalid URL
+		Description: "'%s' volume name is not valid. Please check https://cloud.ibm.com/docs/infrastructure/vpc?topic=vpc-rias-error-messages#volume_id_invalid",
```

Comment:
Can we remove this URL from description and just put text message " Invalid volume name " 
Its confusing to consumer

Reply:
@prgavali , This is TODO~ I already asked VPC team to share more details although I have not seen any error code, once we will get that details then we can update if we remove it then we may skip thats why I kept invalid_id URL